src/DetailsView/components/load-assessment-helper.ts

```diff
@@ -1,34 +1,40 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { AssessmentDataParser } from 'common/assessment-data-parser';
+import { PersistedTabInfo } from 'common/types/store-data/assessment-result-data';
 import { DetailsViewActionMessageCreator } from 'DetailsView/actions/details-view-action-message-creator';
 
-export class LoadAssessmentHelper {
-    constructor(
-        private readonly assessmentDataParser: AssessmentDataParser,
-        private readonly detailsViewActionMessageCreator: DetailsViewActionMessageCreator,
-        private readonly fileReader: FileReader,
-        private readonly document: Document,
-        private readonly tabId: number,
-    ) {}
+export function getAssessmentForLoad(
```

Comment:
I think my suggestion here was more so to _go back_ to what we previously had with load-assessment-helper. Specifically you can create the class with the non-dynamic parameters and then the latter can be passed when the method is called.

This would also avoid adding a static dependency as this currently does to LoadAssessmentButton.